sdk/maps/azure-maps-search/azure/maps/search/_search_client.py

```diff
@@ -65,1050 +45,348 @@ def __init__(
             credential=credential, **kwargs
```

Comment:
Please fully expand and type hint the keyword arguments of the constructor.

Reply:
Also - why are we customizing this?
Can we not just directly expose the generated code? (with a patch for the auth policy if necessary)